package.json

```diff
@@ -44,14 +47,14 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^8.20.0",
+    "eslint": "^7.32.0 || ^8.2.0",
```

Comment:
@Vishalguptag - Please add pre-commit command's in package.json file.

Reply:
okkk sir